providers/github.go

```diff
@@ -31,7 +32,8 @@ var _ Provider = (*GitHubProvider)(nil)
 
 const (
 	githubProviderName = "GitHub"
-	githubDefaultScope = "user:email"
+	githubDefaultScope = "user:email read:org"
```

Comment:
Any reason why this is now the default? Wouldn't this be a breaking change?

Reply:
As it is adding and not removing a scope to the default it is not a breaking change. It was already necessary to add `read:org` when you wanted to get information about the teams and orgs of a user.

Reply:
It's a breaking change if/when a user specified the scope. For example, I was using this flag `--scope=user:email` and this broke things after updating to `7.6.0`. To fix this, I had to either remove my manual setting of the flag or add the new setting `user:email read:org`.

- https://github.com/etho201/docker-pi-stacks/commit/b16eb1b87e83324b745b7cba2a713ba347513e86
- https://github.com/etho201/docker-pi-stacks/commit/22b1d0f18e787fc9a09997089a123091ce550600

Reply:
@etho201 I'm sorry to hear that you faced this issue. I thought we specifically tested this scenario. Can you share the error you encountered and your full configuration?